specification/purview/Azure.Analytics.Purview.DataMap/routes.tsp

```diff
@@ -0,0 +1,2355 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-client-generator-core";
+import "@typespec/http";
+import "@typespec/versioning";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+using Azure.Core.Traits.Private;
+using Azure.ClientGenerator.Core;
+
+namespace PurviewDataMap;
+
+@doc("The ApiVersion query parameter for Atlas API.")
+model AtlasApiVersionParameter {
+  @doc("The API version to use for this operation.")
+  @query("api-version")
+  @minLength(1)
+  apiVersion?: string;
+}
+
+alias AtlasApiVersionParameterTraits = Azure.Core.Traits.VersionParameterTrait<AtlasApiVersionParameter>;
+
+#suppress "@azure-tools/typespec-azure-core/operation-missing-api-version" "This is Atlas API, which does not require api version"
+@doc("The operation defined for Atlas APIs")
+op AtlasOperation<
+  TParams extends TypeSpec.Reflection.Model,
+  TResponse,
+  Traits extends TypeSpec.Reflection.Model = {},
+  TErrorResponse = Azure.Core.Foundations.ErrorResponse
+>(
+  ...TraitProperties<Traits, TraitLocation.ApiVersionParameter>,
+  ...TParams,
+): TResponse | TErrorResponse;
+
+#suppress "@azure-tools/typespec-azure-core/operation-missing-api-version" "This is Atlas API, which does not require api version"
+op AtlasRpcOperation<
+  TParams extends TypeSpec.Reflection.Model,
+  TResponse extends TypeSpec.Reflection.Model,
+  Traits extends TypeSpec.Reflection.Model = {},
+  TErrorResponse = Azure.Core.Foundations.ErrorResponse,
+  TraitContexts extends TraitContext = TraitContext.Undefined
+> is AtlasOperation<
+  TParams & TraitProperties<Traits, TraitLocation.Parameters>,
+  TResponse & TraitProperties<Traits, TraitLocation.Response, TraitContexts>,
+  Traits,
+  TErrorResponse
+>;
+
+model OkResponse<T> {
```

Comment:
Why do we need this model? Can we at least make it an alias?

Reply:
Make it to alias. Thanks.
We need to define the status code of an operation.